test/compress/pure_funcs.js

```diff
@@ -11,7 +11,6 @@ array: {
     }
     expect: {
         function f(a, b) {
-            Math.floor(c / b);
```

Comment:
Shouldn't the expected result be the following?
```
function f(a, b) { 
    c;
}
```
Since `c` is global then its access could throw if not defined. 
```
$ cat foo.js
"use strict";
try {
    c;
} catch (x) {
    console.log("caught exception: ", x);
}
```
```
$ node ./foo.js
caught exception:  ReferenceError: c is not defined
```

Just confirming - in ECMAScript there are no "divide by zero" exceptions, correct? It would produce Infinity or NaN or whatever so no issue removing the divisions here.